paper-tooltip.html

```diff
@@ -122,6 +122,15 @@
         },
 
         /**
+         * Set this to true if you want to manually control when the tooltip
+         * is shown or hidden.
+         */
+        preventTouchEvents: {
```

Comment:
The language here is very close to `preventDefault` and confused me a little. Also, there don't seem to be any touch events at play in the implementation below. Is there a better name for this property?